keylime/tpm/tpm_abstract.py

```diff
@@ -307,6 +342,13 @@ def check_pcrs(self, agent_id, tpm_policy, pcrs, data, virtual, ima_measurement_
         if len(missing) > 0:
             logger.error("%sPCRs specified in policy not in quote: %s" % (("", "v")[virtual], missing))
             return False
+
+        if mb_refstate not in (False, None) and mb_measurement_list not in (False, None):
```

Comment:
Same comment about boolean tests

Reply:
there was reason for this (we discussed the possibility of admitting an empty dictionary as a reference state), but decided against if for this PR